Library/Formula/tasksh.rb

```diff
@@ -0,0 +1,17 @@
+class Tasksh < Formula
+  homepage "http://tasktools.org/projects/tasksh.html"
+  url "http://taskwarrior.org/download/tasksh-1.0.0.tar.gz"
+  head "https://git.tasktools.org/scm/ex/tasksh.git"
+  sha256 "9accc81f5ae3a985e33be728d56aba0401889d21f856cd94734d73c221bf8652"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/tasksh", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!